perma_web/conftest.py

```diff
@@ -401,6 +437,19 @@ def user_with_links_this_month_before_the_15th(link_user, link_factory):
     return link_user
 
 
+@pytest.fixture
+def complex_user_with_bonus_link(link_user_factory, folder_factory, organization, registrar,
+                                 sponsorship_factory, link_factory, in_subfolder=False):
```

Comment:
Nice!

I have a couple questions about this fixture; maybe let's chat?

Reply:
sponsorship_factory(registrar=registrar, user=user, created_by=registrar.user)
The created by user has to be the registrar user--this won't work, use debugger to figure out how to access registrar user from registrar.

Reply:
folder_factory for sponsored folder has to be gotten through sponsorship factory.